src/12_scopes.py

```diff
@@ -19,6 +21,8 @@ def outer():
     y = 120
 
     def inner():
+        nonlocal y
+        # checked out documentation but not quite sure why this works *shrugs*
         y = 999
 
```

Comment:
Well done here. `nonlocal` is used to access variables inside nested functions which allows us to assign values to a variable in an outer, but non-global scope. Here we wanted to change the `print()` to be 999, so we used `nonlocal` to "tie" the re-assignment of y in `inner()` to the scope of `outer()`.